scripts/add-to-historical-roster.py

```diff
@@ -1,11 +1,12 @@
+#!/usr/bin/env python3
 from pathlib import Path
 import pandas as pd
 import click
 
 from pandas import DataFrame
 
 DEFAULT_HISTORICAL = (
-    Path(__file__).parents[1] / "seed" / "Seattle-WA-Police-Department_Historical.csv"
+    Path(__file__).parents[1] / "db" / "seed" / "Seattle-WA-Police-Department_Historical.csv"
```

Comment:
I think this means we'll need to merge #49 in before we merge this one in right?

Reply:
I'm not sure, why would we need to?

Reply:
Oh, no never mind I think it'll be ok now that I look at it. I thought this line was changing a target directory that might have gotten restructured by the integration testing re-write.